sklearnex/linear_model/tests/test_logreg.py

```diff
@@ -15,14 +15,58 @@
 # ===============================================================================
 
 import numpy as np
+import pytest
 from numpy.testing import assert_allclose
-from sklearn.datasets import load_iris
+from sklearn.datasets import load_breast_cancer, load_iris
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
 
+from daal4py.sklearn._utils import daal_check_version
+from onedal.tests.utils._dataframes_support import (
+    _as_numpy,
+    _convert_to_dataframe,
+    get_dataframes_and_queues,
+)
+from sklearnex import config_context
 
-def test_sklearnex_import():
+
+def test_sklearnex_import_cpu():

```

Comment:
Answer to the first question: we only support binary classification with newton-cg solver for GPU while for CPU we support multiclass classification and also lbfgs solver. That's why in case of CPU we would need more tests so it's seems reasonable to make several tests. Nevertheless, we can use different naming, for example, test_binary and test_multiclass